README.md

```diff
@@ -93,3 +93,11 @@ typed enums in places where the underlying provider may only have weakly typed s
 To do this, first add the files in the appropriate package sub-directory of the sdk, and then add the requisite directives to the
 provider file.  See the [AWS overlays section in resources.go](https://github.com/pulumi/pulumi-aws/blob/master/provider/resources.go#L4486) for
 an example of this in action.
+
+# tfgen options
+
+tfgen, the command that generates Pulumi schema/code for a bridged provider supports the following environment variables:
+
+`PULUMI_MISSING_MAPPING_ERROR`: If truthy, fail if a data source/resource in the TF provider is not mapped to the Pulumi provider.
+`PULUMI_MISSING_DOCS_ERROR`: If truthy, fail if docs cannot be found for a data source/resource.
+`PULUMI_EXTRA_MAPPING_ERROR`: If truthy, fail if a mapped resource does not exist in the TF provider.
```

Comment:
likewise as above, is this in preparation for future not-quite-errors?

Reply:
The docs flag is new, but the other two were just undocumented.